tests/framework/utils/k8s_helper.go

```diff
@@ -258,7 +258,7 @@ func (k8sh *K8sHelper) DeleteResource(args ...string) error {
 func (k8sh *K8sHelper) WaitForCustomResourceDeletion(namespace, name string, checkerFunc func() error) error {
 
 	// wait for the operator to finalize and delete the CRD
-	for i := 0; i < 60; i++ {
+	for i := 0; i < 90; i++ {
```

Comment:
Since each sleep is 2s, we're going from a 2min wait to 3min? Seems fine to wait longer, though would be nice to understand what is taking so long during deletion.

Reply:
I'm not sure. Comparing the operator log to the test log, the filesystem just needed like 3-4 seconds longer.

Reply:
And it's the filesystem deletion that needs the extra time, not object.